### DISCUSSION TITLE
Cant create mex file

### DISCUSSION THREAD

**Discussion by NOTaShinigami** [JSBSim-Team/jsbsim#780] [Q&A]
I'm attempting to build the SFunction, as instructed in the readme. Since I need JSBSim.lib to be produce the mex file, I tried building using Microsoft Visual Studio, but my build never succeeded; I received the message: "Build: 2 successful, 1 failed, 0 up-to-date, 0 skipped ===================================================== Build failed.
As a result, I attempted to create the .lib files by changing the -DCMAKE_GNUtoMS variable to ON, but this too failed, and I received .a files.
I attempted to generate the mex file by changing the Matlab compiler, but I received the error wsock32.lib not found for MinGW and the error JSBSim.lib file not found for Visual C++.

**Comment by bcoconni** [comment]
Could you please provide the full error message ?

**Reply by NOTaShinigami** [reply]
It's alright. I was able to get around the issue by generating the files using the GUI. Perhaps my generation command was incorrect, causing problems for Microsoft Visual Studio.
